### PR TITLE
Issue/2124 button group

### PR DESCRIPTION
**Pull request by tatjana-gartner2**
### Pull Request Checklist

- [x] I have read and accept the [Contributor License Agreement](https://github.com/porscheofficial/oss-docs/blob/67c5b24838a293ce7a964884e6005eb71f2b8579/CONTRIBUTOR_LICENSE_AGREEMENT.md)

#### References

- Preview: https://designsystem.porsche.com/issue/.../

#### Scope

Short description of the pull request scope

### CODE REVIEW

File: packages/components/src/components/button-group/button-group-styles.ts [hunk 1]
```diff
@@ -47,13 +27,10 @@ export const getComponentCss = (direction: BreakpointCustomizable<ButtonGroupDir
       },
       div: {
         display: 'flex',
-        marginTop: `-${spacingStaticMedium}`,
+        marginTop: `${spacingFluidSmall}`,
```
Comment (denyo) [inline packages/components/src/components/button-group/button-group-styles.ts:30]: String interpolation is useless.
I think something is wrong now, because this adds a `margin-top` to the component where there wasn't any in the past.

**Comment by denyo** [comment]
Some icons / wordings are missing in VRTs.
Bildschirm­foto 2023-02-20 um 10 00 15

**Comment by denyo** [comment]
What's going on with the alignment here?

Bildschirm­foto 2023-02-20 um 09 59 35

File: packages/components/src/components/button-group/button-group-styles.ts (hunk 1 above)
Comment (tatjana-gartner2) [reply packages/components/src/components/button-group/button-group-styles.ts:30]: In the past the margin was removed from the button group wrapper and added to the button itself. This also increases the distance between all buttons when direction="row" is selected. I didn't see any need for the margin to be on the buttons themselves, so I removed the margin from the buttons and set it to the button group.

The spacing is bigger now because of using the fluid spacing (discussed this with johannes on friday)

Comment (stephanschroeter) [reply packages/components/src/components/button-group/button-group-styles.ts:31]: I also don't think that marginTop is needed at all since, everything is now handled with flex gap.

**Review by denyo — APPROVED** [review]

File: packages/components/CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Background color of `Scroller`'s `prev` and `next` buttons in dark theme
 - Partials: Removed deprecated `withoutTags` option for all partials, please use `format: 'jsx'` instead
 - `Divider` matches new design language.
+- `Button Group` matches new design language.
```
Comment (denyo) [inline packages/components/CHANGELOG.md:17]: Bad merge? Should be combined with line above.

File: packages/components/CHANGELOG.md [hunk 3]
```diff
@@ -405,21 +406,6 @@ to the new values since those ones will be removed with next major version.
 + …addMessage({ text: `Some message`, state: 'info' })
 ```
 
-##### Divider:
-
```
Comment (denyo) [inline packages/components/CHANGELOG.md:409]: Why was this removed?

**Review by denyo — CHANGES_REQUESTED** [review]
See comments.

**Review by denyo — APPROVED** [review]
